usermods/usermod_v2_animartrix/usermod_v2_animartrix.h

```diff
@@ -7,58 +7,58 @@
-static const char _data_FX_mode_Complex_Kaleido_5[] PROGMEM = "💡Complex_Kaleido_5 ☾@;;1";
-static const char _data_FX_mode_Complex_Kaleido_4[] PROGMEM = "💡Complex_Kaleido_4 ☾@;;1";
-static const char _data_FX_mode_Complex_Kaleido_3[] PROGMEM = "💡Complex_Kaleido_3 ☾@;;1";
-static const char _data_FX_mode_Complex_Kaleido_2[] PROGMEM = "💡Complex_Kaleido_2 ☾@;;1";
-static const char _data_FX_mode_Complex_Kaleido[] PROGMEM = "💡Complex_Kaleido ☾@;;1";
-static const char _data_FX_mode_SM10[] PROGMEM = "💡SM10 ☾@;;1";
-static const char _data_FX_mode_SM9[] PROGMEM = "💡SM9 ☾@;;1";
-static const char _data_FX_mode_SM8[] PROGMEM = "💡SM8 ☾@;;1";
-static const char _data_FX_mode_SM7[] PROGMEM = "💡SM7 ☾@;;1";
-static const char _data_FX_mode_SM6[] PROGMEM = "💡SM6 ☾@;;1";
-static const char _data_FX_mode_SM5[] PROGMEM = "💡SM5 ☾@;;1";
-static const char _data_FX_mode_SM4[] PROGMEM = "💡SM4 ☾@;;1";
-static const char _data_FX_mode_SM3[] PROGMEM = "💡SM3 ☾@;;1";
-static const char _data_FX_mode_SM2[] PROGMEM = "💡SM2 ☾@;;1";
-static const char _data_FX_mode_SM1[] PROGMEM = "💡SM1 ☾@;;1";
-static const char _data_FX_mode_Big_Caleido[] PROGMEM = "💡Big_Caleido ☾@;;1";
-static const char _data_FX_mode_RGB_Blobs5[] PROGMEM = "💡RGB_Blobs5 ☾@;;1";
-static const char _data_FX_mode_RGB_Blobs4[] PROGMEM = "💡RGB_Blobs4 ☾@;;1";
-static const char _data_FX_mode_RGB_Blobs3[] PROGMEM = "💡RGB_Blobs3 ☾@;;1";
-static const char _data_FX_mode_RGB_Blobs2[] PROGMEM = "💡RGB_Blobs2 ☾@;;1";
-static const char _data_FX_mode_RGB_Blobs[] PROGMEM = "💡RGB_Blobs ☾@;;1";
-static const char _data_FX_mode_Polar_Waves[] PROGMEM = "💡Polar_Waves ☾@;;1";
-static const char _data_FX_mode_Slow_Fade[] PROGMEM = "💡Slow_Fade ☾@;;1";
-static const char _data_FX_mode_Zoom[] PROGMEM = "💡Zoom ☾@;;1";
-static const char _data_FX_mode_Hot_Blob[] PROGMEM = "💡Hot_Blob ☾@;;1";
-static const char _data_FX_mode_Spiralus2[] PROGMEM = "💡Spiralus2 ☾@;;1";
-static const char _data_FX_mode_Spiralus[] PROGMEM = "💡Spiralus ☾@;;1";
-static const char _data_FX_mode_Yves[] PROGMEM = "💡Yves ☾@;;1";
-static const char _data_FX_mode_Scaledemo1[] PROGMEM = "💡Scaledemo1 ☾@;;1";
-static const char _data_FX_mode_Lava1[] PROGMEM = "💡Lava1 ☾@;;1";
-static const char _data_FX_mode_Caleido3[] PROGMEM = "💡Caleido3 ☾@;;1";
-static const char _data_FX_mode_Caleido2[] PROGMEM = "💡Caleido2 ☾@;;1";
-static const char _data_FX_mode_Caleido1[] PROGMEM = "💡Caleido1 ☾@;;1";
-static const char _data_FX_mode_Distance_Experiment[] PROGMEM = "💡Distance_Experiment ☾@;;1";
-static const char _data_FX_mode_Center_Field[] PROGMEM = "💡Center_Field ☾@;;1";
-static const char _data_FX_mode_Waves[] PROGMEM = "💡Waves ☾@;;1";
-static const char _data_FX_mode_Chasing_Spirals[] PROGMEM = "💡Chasing_Spirals ☾@;;1";
-static const char _data_FX_mode_Rotating_Blob[] PROGMEM = "💡Rotating_Blob ☾@;;1";
+static const char _data_FX_mode_Module_Experiment10[] PROGMEM = "💡Module_Experiment10 ☾@;;1;2";
```

Comment:
Beware that this will also select "* Random Cycle" palette as a selected palette. (...;1;...)
Use either ! or empty.

Reply:
This different in AC? In MM it shows currently as this effect does not use a palette

Reply:
Why not just test my code rather than make guesses about runtime?

Reply:
`...;1;...` is not technically correct in any case. It may behave as you intend ATM due to wrong check/bug in `if` statement.
If you do not want palettes to be used (and shown in UI) do not assign any character into this field i.e. `...;;...`
This will prevent unwanted behaviour in the future if the `if` statement changes.

I am sorry if this frustrates you and if you do not want to modify the code I am ok with that too.

Reply:
updated